.github/workflows/linux_meson.yml

```diff
@@ -67,7 +67,7 @@ jobs:
         python -m pip install numpy cython pytest pytest-xdist pytest-timeout pybind11 mpmath gmpy2 pythran ninja meson click rich-click doit pydevtool pooch hypothesis
 
     - name: Install Python packages from repositories
-      if: matrix.python-version == '3.12-dev'
+      if: matrix.python-version == '3.12' # these are the runs which would use python dev versions
```

Comment:
likewise here with rephrasing the comment

Reply:
yes, I fixed the grammar